### PR TITLE
Add performance tuning section for RDMA

### PR DESCRIPTION
**Pull request by pizhenwei**
valkey-server is performance sensitive on RDMA, add man page for deeply performance tuning.

[Link](https://github.com/valkey-io/valkey/pull/1209) for valkey changes.

### CODE REVIEW

**Review by enjoy-binbin — APPROVED** [review]

**Review by zuiderkwast — APPROVED** [review]

File: topics/RDMA.md [hunk 1]
```diff
@@ -50,6 +50,47 @@ Or:
...
+```
+VALKEY VALKEY VALKEY VALKEY VALKEY VALKEY     VALKEY
+   |      |      |      |      |      |          |
+ CPU0   CPU1   CPU2   CPU3   CPU4   CPU5  ...  CPUX
+   |      |      |      |      |      |          |
+ INTR0  INTR1  INTR2  INTR3  INTR4  INTR5      INTRX
+```
+
+Use 0 and positive numbers to specify the RDMA completion vector, or specify -1 to allow
+the server to use a random vector for a new connection. The default vector is -1.
```
Comment (madolson) [inline topics/RDMA.md:92]: I commented on the main PR, but I don't follow what specifically this means.

File: topics/RDMA.md [hunk 2]
```diff
@@ -50,6 +50,47 @@ Or:
...
+ INTR0  INTR1  INTR2  INTR3
+```
+
+### Example 2
+
+- Pin hardware interrupt vectors [0, X] to CPU [0, X].
+- Set CPU affinity for valkey to CPU [0, X].
+- Valkey server [M] uses RDMA completion vector [M].
+
+A single CPU handles hardware interrupts, the RDMA completion queue, and the valkey server.
```
Comment (madolson) [inline topics/RDMA.md:80]:
```suggestion
All CPUs handles hardware interrupts, the RDMA completion queue, and the valkey server.
```
Unless I'm missing something.

**Comment by pizhenwei** [comment]
@madolson Add link for `ibv_create_cq` and describe `1:1:1` pinning, please take a look. Thanks!

**Review by madolson — APPROVED** [review]

**Comment by madolson** [comment]
Thanks, this made more sense to me :)

**Pull request merged**